### PR TITLE
export xdpi and ydpi for android

### PR DESCRIPTION
**Pull request by rohanhc**

### CODE REVIEW

**Comment by jishnu7** [comment]
LGTM

**Comment by rampr** [comment]
Looks Good to me too

**Comment by mit456** [comment]
Approved

File: TeaLeaf/jni/platform/navigator.h [hunk 1]
```diff
@@ -17,6 +17,8 @@
 
 typedef struct navigator_info_t {
     int density_dpi;
+    int xdpi;
+    int ydpi;
```
Comment (jishnu7) [inline TeaLeaf/jni/platform/navigator.h]: these should be float, right?

Comment (rohanhc) [inline TeaLeaf/jni/platform/navigator.h]: I thought int was fine? But yes, output from java is float. I'll change it.

Comment (rohanhc) [inline TeaLeaf/jni/platform/navigator.h]: Done

**Pull request merged**